msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AcquireTokenSilentSupplier.java

```diff
@@ -113,15 +119,15 @@ private boolean shouldRefresh(SilentParameters parameters, AuthenticationResult
 
         //If forceRefresh is true, no reason to check any other option
         if (parameters.forceRefresh()) {
-            setCacheTelemetry(CacheTelemetry.REFRESH_FORCE_REFRESH.telemetryValue);
+            setCacheTelemetry(CacheRefreshReason.FORCE_REFRESH);
             log.debug("Refreshing access token because forceRefresh parameter is true.");
             return true;
         }
 
         //If the request contains claims then the token should be refreshed, to ensure that the returned token has the correct claims
         //  Note: these are the types of claims found in (for example) a claims challenge, and do not include client capabilities
         if (parameters.claims() != null) {
-            setCacheTelemetry(CacheTelemetry.REFRESH_FORCE_REFRESH.telemetryValue);
+            setCacheTelemetry(CacheRefreshReason.CLAIMS);
             log.debug("Refreshing access token because the claims parameter is not null.");
```

Comment:
```suggestion
            log.debug("Refreshing access token due to reason: " + CacheRefreshReason.CLAIMS);
```

Reply:
In the latest commit I improve these logs, now is uses proper String formatting and includes the enum value like you suggested.